Formula/l/litmusctl.rb

```diff
@@ -0,0 +1,29 @@
+class Litmusctl < Formula
+  desc "Command-line interface for interacting with LitmusChaos"
+  homepage "https://litmuschaos.io"
+  url "https://github.com/litmuschaos/litmusctl/archive/refs/tags/1.5.0.tar.gz"
+  sha256 "363e58ecdf22dc9311bffde69e6f69a89bb30d77189791af9128f28e730b6ea6"
+  license "Apache-2.0"
+
+  head "https://github.com/litmuschaos/litmusctl.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    # Debug message
+    ohai "Building litmusctl..."
+    # Build the binary
+    ldflags = %W[
+      -s -w
+      -X main.CLIVersion=#{version}
+    ]
+    system "go", "build", *std_go_args(ldflags:), "-o", bin/"litmusctl"
```

Comment:
This is not needed since it defaults to the formula name
```suggestion
    system "go", "build", *std_go_args(ldflags:)
```